docs/atomic-diff.1.md

```diff
@@ -63,6 +69,10 @@ Compare the files and RPMs (without versions) in images 'foo1' and 'foo2' and ou
 Compare only the metadata between images 'foo1' and 'foo2'
 
     atomic diff -m foo1 foo2
+    
+Compare files by 'sha256digests' and 'time' between images 'foo1' and 'foo2'
+
+    atomic diff foo1 foo2 --keywords sha256digest time
```

Comment:
I think the handling of multiple keywords does not work well.  What happens if I do


atomic diff --keywords sha256digest time foo1 foo2

Does it work
Or if I had a container named time.

atomic diff --keywords sha256digest time foo1